src/main/java/tech/sourced/siva/Header.java

```diff
@@ -4,31 +4,55 @@
 import java.nio.file.attribute.PosixFilePermission;
 import java.util.Set;
 
+/**
+ * Header contains the meta information from a file that a single {@link IndexEntry} keeps in the siva file.
+ */
 public class Header {
     private final String name;
     private final FileTime modificationTime;
     private final Set<PosixFilePermission> fileMode;
     private final Flag flag;
 
+    /**
+     * Builds a Header object with the given file meta information(name, UNIX modification
+     * time, UNIX file mode and Flag).
+     *
+     * @param name             Entry name.
+     * @param modificationTime Modification time as UNIX time in nanoseconds.
```

Comment:
In JavaDoc, formatting the `@param` like this and below is not a standard practice.

Although presumably this aids readability, but here is [some background](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment) why this might be not good for you in the long term.

Reply:
As with imports issue, code formatting is done by intellij too, should I also change it?

Reply:
+1 IDK why is that a default but remember doing this https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports before